run_dir/static/js/element_flowcell.js

```diff
@@ -0,0 +1,1374 @@
+
+const vElementApp = {
+    data() {
+        return {
+            ngi_run_id: "some_id",
```

Comment:
Couldn't this be empty instead of `some_id`?

Reply:
Yes, adjusted.